docs/community/devstack.md

```diff
@@ -0,0 +1,139 @@
+
+
+### api/v3
+
+Found in https://github.com/ansible/galaxy_ng/tree/master/galaxy_ng/app/api/v3
+
+The v3 api is 100% focused on collections and collection namespaces. As time progresses, the v3 api becomes more and more a redirect to endpoints provided by pulp_ansible. 
+
+Access control over collections is controlled by the v3 collection namespaces and pulp object level RBAC. Users are added to a group and the group is bound to the namespace with a permission. For the community instances, the group and permission bindings are handled automatically by the social auth handler.
+
+## Docker compose
+
+For development work on the galaxy_ng project or when integration testing a pulp_ansible+galaxy_ng feature, docker-compose is the preferred pathway. With all the moving parts in the stack, docker-compose is the simplest way to spin up a functional dev environment.
+
+!!! warning
+    docker-compose 2.x (the golang rewrite) has compatibility issues with our current stack, so please use the 1.x version that was written in python.
+
+### Profiles & Configuration
+
+The galaxy_ng project is a highly configurable application, but only at startup time. Almost all of the behavior of the application is influenced by environment variables and static settings.py files that are read in by [django](https://www.djangoproject.com/) and [dynaconf](https://www.dynaconf.com/). None of these settings live in the database currently, so they can only be declared before the application starts.
+
+The "community" profile of galaxy_ng is a combination of settings driven by the docker-compose files found in [the standalone-community directory](https://github.com/ansible/galaxy_ng/tree/master/dev/standalone-community). To get his profile to run, you must do a few things:
+
+1. Copy the ".compose.env.example" file at the root of the repository to a new file named ".compose.env".
+2. Change the COMPOSE_PROFILE value in ".compose.env" from "standalone" to "standalone-community".
+3. Add SOCIAL_AUTH_GITHUB_KEY=<value> to the bottom of ".compose.env".
+4. Add SOCIAL_AUTH_GITHUB_SECRET=<value> to the bottom of ".compose.env".
+5. Add SOCIAL_AUTH_LOGIN_REDIRECT_URL = "/" to the bottom of galaxy_ng/app/settings.py
+6. Add SOCIAL_AUTH_REDIRECT_IS_HTTPS = True to the bottom of galaxy_ng/app/settings.py if your stack will be behind an http server with SSL enabled.
+
+The github key and secret are provided by configuring an oauth account on github.com under your user's developer settings. The oauth application config needs to have a valid+resolvable homepage url and the callback url should be the same host with "/complete/github/" as the path. For local testing, you can use something like http://localhost:8002 as the host.
```

Comment:
Maybe "valid and resolvable" instead of "valid+resolvable" ?

Reply:
"something like" is probably not needed.

For local testing, you can use http://localhost:8002 as the host.